windows/CodePush.cs

```diff
@@ -0,0 +1,668 @@
+﻿using System;
+using System.Collections.Generic;
+using ReactNative.Bridge;
+using ReactNative.Modules.Core;
+using ReactNative.UIManager;
+using Windows.ApplicationModel;
+using Windows.Storage;
+using System.IO;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using System.Reflection;
+using Windows.Web.Http;
+using Windows.Storage.FileProperties;
+using ReactNative;
+
+namespace CodePush.ReactNative
+{
+    public class CodePushModule : IReactPackage
```

Comment:
We should mark this class as sealed, and I'm actually wondering whether we should call it "CodePushPackage" since that more accurately reflects its purpose to the app developer.